### ISSUE TITLE
Webapp-runner deployment does not allow docker volume file mounts 

### ISSUE THREAD

**Issue by pvannierop** [cBioPortal/cbioportal#6359]
While using the new docker solution for portal deployment we came across the limitation of  that the new solution, that runs the .war file with webapp-runner is not compatible with file mounts in docker. Placing a custom logo in the /image directory of the expanded webapp-runner source dir prior to running webapp-runner interferes with server startup.
```
  cbioportal-container:
    container_name: cbioportal-container
    image: cbioportal/cbioportal:master
    ports:
    - "8081:8080"
    volumes:
    # by default webapp-runner decompresses the war into /expanded prior to serving the content
    # line below does not work it interfered with the web-runner unpacking
    - $PWD/custom_logo.jpg:/target/tomcat.8080/webapps/expanded/images/custom_logo.jpg:ro
    - $PWD/portal.properties:/cbioportal/portal.properties:ro
    - $PWD/frontendConfig.json:/tmp/frontendConfig.json:ro
    environment:
      CATALINA_OPTS: "-Xms2G -Xmx24G"
    command: /bin/sh -c 'java -Dauthenticate=false -jar webapp-runner.jar /app.war'
    restart: always
    depends_on:
      - cbioDB
```
Is there an alternative way of deployment that allows unpacking and deployment via Tomcat in the same image?

**Comment by pvannierop** [comment]
@inodb @n1zea144 @fedde-s 
I have solved this issue in [PR #6369](https://github.com/cBioPortal/cbioportal/pull/6369). Please review and merge :)

**Comment by inodb** [comment]
this has been fixed
